playwright.config.js

```diff
@@ -43,27 +43,27 @@ export default defineConfig({
         locale: 'en-EN'
       },
       dependencies: ['setup']
-    },
+    }
 
-    {
-      name: 'firefox',
-      use: {
-        browserName: 'firefox',
-        locale: 'en-EN'
-      },
-      dependencies: ['setup']
-    },
+    // {
+    //   name: 'firefox',
+    //   use: {
```

Comment:
This can be removed, but will be implemented again in the near future, so i thought it would be better to just comment out the code. Do you prefer to remove it?

Reply:
I think it's ok like this, if the plan is to enable these browsers again in the future